src/buildercore/core.py

```diff
@@ -111,13 +111,23 @@ def connect_aws_with_stack(stackname, service):
     return connect_aws_with_pname(pname, service)
 
 def find_ec2_instance(stackname):
-    "returns ec2 instance data for a *specific* stackname"
+    "returns list of ec2 instances data for a *specific* stackname"
     # filters: http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html
-    kwargs = {
+    filter_by_cluster = {
```

Comment:
I'm not sure what is the style convention here for multi-line nested dictionaries, I am following what is already there